<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Przebieg_zmiennosci</class>
 <widget class="QMainWindow" name="Przebieg_zmiennosci">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Przebieg zmienności funkcji</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: Arial;
	font-size: 24px;
}
#centralwidget{
	background-color: #0ca678;
}
QPushButton{
	height: 60px;
	border: 2px solid white;
	border-radius: 12px;
	color: white;
	background-color: transparent;
}
QPushButton:hover{
background-color: white; 
color: #0ca678;
}
QLabel{
	padding-right: 10px;
	border: 2px solid white;
	border-radius: 12px;
	color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>571</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="klawiatura">
     <property name="horizontalSpacing">
      <number>7</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn3">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn2">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btn1">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="btn0">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btnSin">
       <property name="text">
        <string>sin</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="btnCos">
       <property name="text">
        <string>cos</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="btnPlus">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="btnParClose">
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btnMinus">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="btnMult">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="btnDiv">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="btnDot">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="btnPow">
       <property name="text">
        <string>^</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="btnSqrt">
       <property name="text">
        <string>√</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="btnParOpen">
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="btn5">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btn7">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn6">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btn4">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn8">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnInf">
       <property name="toolTip">
        <string>∞</string>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;Myanmar Text&quot;;
font-size: 23px;</string>
       </property>
       <property name="text">
        <string>∞</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btnX">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btn9">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btnPi">
       <property name="text">
        <string>π</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
border-radius: 0;</string>
    </property>
    <property name="text">
     <string>Wróć</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRun">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: left;
color: white;
</string>
    </property>
    <property name="midLineWidth">
     <number>-4</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAllClear">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelError">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
